pkg/apis/pipeline/v1beta1/pipelinerun_conversion_test.go

```diff
@@ -263,6 +402,105 @@ func TestPipelineRunConversionFromDeprecated(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "taskRuns",
```

Comment:
I would think the desired output here depends on the value of the embedded-status feature flag

Reply:
Put in the related #5813, this shall be resolved with the decision made in #5813 about the handling of the full `embedded-status` in v1.

Reply:
Same as above: I think this does need to be addressed in this PR, but we can just fail the conversion if the feature flag is set to the incorrect value. The TODO is for creating a better solution in those cases.

Reply:
Thanks Lee. That makes sense, also updated the issue.